Frontend/src/Components/CreateListing/InfoSection.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+
+const InfoSection = ({ category, setCategory, brand, setBrand, condition, setCondition }) => {
+  return (
+    <div className="infoSection">
+      <h3>Info</h3>
+      <label>
+        Category
+        <select value={category} onChange={(e) => setCategory(e.target.value)}>
+            <option value="" disabled>Select a category</option>
+            <option value="clothing">Clothing</option>
+            <option value="accessories">Accessories</option>
+            <option value="accessories">Furniture</option>
```

Comment:
are these values all supposed to be accessories?

Reply:
No, but ive made the changes!